cmd/caddy/main.go

```diff
@@ -34,5 +34,5 @@ import (
 )
 
 func main() {
-	caddycmd.Main()
+	caddycmd.Execute()
```

Comment:
Leaving a note for self when I come back to this later: changing Main to Execute this breaks xcaddy because the template uses `caddycmd.Main()`. That said, `.Execute()` doesn't have to be `.Execute()`. Whatever `.Execute()` does, `.Main()` can do, and we just keep `.Main()` and not have `.Execute()`.